### DISCUSSION TITLE
Neovim everywhere

### DISCUSSION THREAD

**Discussion by tesses** [pit-ray/win-vind#105] [General]
I was viewing the roadmap of neovim when i noticed that they are planning to add an api for embedded editor's integrations "everywhere" 

https://www.reddit.com/r/neovim/comments/vbb3va/neovim_roadmap_embed_neovim_everywhere/

I need to find more informations.

Probably this feature can help the developing of the tool.
What do you think?

**Comment by pit-ray** [comment]
I did not know this because I do not use neovim and only use pure Vim.

It would be interesting to have a [firenvim](https://github.com/glacambre/firenvim)-like feature that can also handle binary content like MS Word.
